application/views/interface_assets/footer.php

```diff
@@ -1084,6 +1084,22 @@ function searchButtonPress(){
 	});
 <?php } ?>
 
+	$('#stationProfile').change(function() {
+		var stationProfile = $('#stationProfile').val();
+      console.log("TEST "+stationProfile);
```

Comment:
Remove this line first :)

Reply:
That was just a test if you are paying attention :D

Reply:
:)